CHANGES.md

```diff
@@ -84,6 +84,7 @@ and the first release covered by our new stability policy.
 - Change HTML theme to Furo primarily for its responsive design and mobile support
   (#2793)
 - Deprecate the `black-primer` tool (#2809)
+- Document Python support policy (#2819)
```

Comment:
Arguably this could count as a highlight

Reply:
I hesitated, because it doesn't really affect anything currently 😄 but I can change it if you'd like

Reply:
Can't hurt to have ti here ... might make more people come read it advertising it here ...

Reply:
Not sure if I misunderstood you Cooper, but I think Richard was asking if it should be put in the "highlights" section at the very top instead